src/styles/price.scss

```diff
@@ -0,0 +1,20 @@
+.price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 16px 16px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use multiples of 4px for margin and padding. Please adjust the margin values accordingly.